container/container_windows.go

```diff
@@ -10,6 +10,10 @@ import (
 	containertypes "github.com/docker/docker/api/types/container"
 )
 
+const (
+	containerSecretMountPath = `C:\ProgramData\Docker\secrets`
```

Comment:
I would prefer a less confusing location, as this collides with the daemon directory. That is not be an issue, really, except for Docker in Docker (which is not supported on Windows yet), but I am open to suggestions.